rdagent/scenarios/data_science/proposal/exp_gen/proposal.py

```diff
@@ -879,6 +888,22 @@ def gen(
             )
 
         # Step 2: Propose hypothesis based on the identified problems (and sampled ideas)
+        if self.scen.is_finetune:
```

Comment:
I prefer to place it into an individual proposal instead of bring complexity to current code